libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/metastore/DomainMetaStoreTest.java

```diff
@@ -41,6 +44,8 @@ public void testDomainMetaStoreDefaults() {
         assertTrue(metaStore.isValidBusinessService("athenz", "security"));
         assertTrue(metaStore.isValidBusinessService("athenz", null));
 
+        assertEquals(metaStore.getValidBusinessServices(null), new ArrayList<>());
```

Comment:
we should add one of those for other attributes as well for code coverage.

Reply:
Added missing calls